pkg/hive/install.go

```diff
@@ -193,6 +193,10 @@ func (c *clusterManager) clusterDeploymentForInstall(doc *api.OpenShiftClusterDo
 			Annotations: map[string]string{
 				"hive.openshift.io/try-install-once":                "true",
```

Comment:
(unrelated/latent) Suggest using `[ClusterDeployment.Spec.InstallAttemptsLimit](https://github.com/openshift/hive/blob/903546686f6109e4c0389d72893cf0272a00e2cb/vendor/github.com/openshift/hive/apis/hive/v1/clusterdeployment_types.go#L192)=1` instead. Does the same thing, but is the official/documented/supported mechanism.